plugins/github-pull-requests-board/src/utils/functions.ts

```diff
@@ -30,6 +32,17 @@ export const getProjectNameFromEntity = (entity: Entity): string => {
   return entity?.metadata.annotations?.[GITHUB_PULL_REQUESTS_ANNOTATION] ?? '';
 };
 
+export const getUserNameFromEntity = (entity: Entity): string => {
+  return entity?.metadata.name ?? '';
```

Comment:
not being fully involved with how this is used, I'll just note that just the name isn't normally enough to identify an entity, all three of the kind, namespace, and name are needed

Reply:
This one is interesting because I need the name to make my Github GraphQL query, and there is no kind/namespace concept on the Github side of things.

Looking at a user entity, it seems like what I'm looking for is `github.com/user-login` in the user metadata, rather than the Backstage entity name 💡 I'll make the change